test/ffc-rules-engine.test.js

```diff
@@ -10,6 +10,7 @@ function getParcelWithDefaults (options) {
     areaFeatures: [],
     previousActions: [],
     sssi: false,
+    landCoverClass: 100,
```

Comment:
I think this should probably default to 0 unless there is a reason to use 100

Reply:
👍  done